src/bsdfs/roughconductor.cpp

```diff
@@ -346,6 +353,10 @@ class RoughConductor final : public BSDF<Float, Spectrum> {
             F = fresnel_conductor(UnpolarizedSpectrum(dot(si.wi, H)), eta_c);
         }
 
+        /* If requested, include the specular reflectance component */
+        if (m_specular_reflectance)
+            F *= unpolarized<Spectrum>(m_specular_reflectance->eval(si, active));
```

Comment:
Change `result` above to type `UnpolarizedSpectrum` & change this line to ` result *= m_specular_reflectance->eval(si, active);
`.